test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
```

Comment:
@dboltovskyi, please ask BA to add in attached specification (..._Specification_New External UCS-related params_v1.docx) the requirement: "[Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_off" struct"

Also link for External User Consent Settings (EXTERNAL_PROPRIETARY flow) - https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=307571424 is not working.